pkg/volume/emptydir/empty_dir.go

```diff
@@ -294,12 +295,17 @@ func (ed *emptyDir) assignQuota(dir string, mounterSize *resource.Quantity) erro
 	if mounterSize != nil {
 		// Deliberately shadow the outer use of err as noted
 		// above.
-		hasQuotas, err := fsquota.SupportsQuotas(ed.mounter, dir)
+		userNamespaceDisabled := true
```

Comment:
maybe this could use a comment saying  something like "host users field is interpreted as true by default, so a pod is by default not confined by a user namespace"
also because of that, I believe this should be `userNamespaceDisabled := false` or just `var userNamespaceDisabled bool`

Reply:
also maybe a link to https://github.com/kubernetes/kubernetes/blob/a8d51f4f0529c671cd8d097d3ba43a478db481fd/pkg/kubelet/userns/userns_manager.go#L389 and a TODO added here and there saying "if the default value for hostUsers ever changes, change this as well" sort of thing?

Reply:
Since the hostUsers being true by default means that the user namespace is disabled, the variable has a default true value. Do you suggest we use a different name?

Reply:
Changed the flagName to UserNamespacesEnabled